docs/UserGuide.md

```diff
@@ -145,10 +145,11 @@ Adds a person to the address book.
 
 Format: `add n/NAME e/EMAIL g/GENDER a/AGE [d/DETAIL] [t/STUDY_GROUP_TAG]…​`
 
-* Email, gender and study group tags are **case-insensitive**
-* Valid entries for gender are M/m/F/f
-* Age must be a positive integer less than 150
-* Two contacts are considered **duplicates** if they share the same **email** address
+* Email, gender and study group tags are **case-insensitive**.
+* Valid entries for names must consist only of letters; numbers and special characters are not allowed. Each part of the name should be separated by exactly one space, with no consecutive spaces permitted.
+* Valid entries for gender are M/m/F/f.
+* Age must be a positive integer less than 150.
```

Comment:
I believe @wkxcass 's pr is making some changes here. The wording is different and probably more specific/correct. Do you want to take a look at #313 and compare?

Reply:
I think best case is I merge my UG pr first, then @wujiaweijoanna can merge any related conflicts afterwards! (Need reviews first hahaha)

Reply:
Update: merged my UG pr!